<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>895</width>
    <height>713</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>11</y>
      <width>491</width>
      <height>226</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="averageLabel">
       <property name="font">
        <font>
         <family>Ubuntu Mono</family>
         <pointsize>25</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Average Reaction Time: 0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="reactionLabel">
       <property name="font">
        <font>
         <family>Ubuntu Mono</family>
         <pointsize>25</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>Reaction Time: 0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="acertosLabel">
       <property name="font">
        <font>
         <family>Ubuntu Mono</family>
         <pointsize>25</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Acertos: 0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="errosLabel">
       <property name="font">
        <font>
         <family>Ubuntu Mono</family>
         <pointsize>25</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Erros: 0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="scoreLabel">
       <property name="font">
        <font>
         <family>Ubuntu Mono</family>
         <pointsize>25</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Score: 0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="resetButton">
       <property name="minimumSize">
        <size>
         <width>289</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Ubuntu Mono</family>
         <pointsize>16</pointsize>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Reset</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>resetButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
